src/Observers/HttpClientObserver.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace LaraDumps\LaraDumps\Observers;
+
+use Illuminate\Http\Client\Events\{RequestSending, ResponseReceived};
+use Illuminate\Http\Client\{Request, Response};
+use Illuminate\Support\Facades\Event;
+use LaraDumps\LaraDumps\LaraDumps;
+use LaraDumps\LaraDumps\Payloads\{Payload, TableV2Payload};
+
+class HttpClientObserver
+{
+    private bool $enabled = false;
+
+    private ?string $label = null;
+
+    private array $trace = [];
+
+    public function register(): void
+    {
+        Event::listen(RequestSending::class, function (RequestSending $event) {
+            if (!$this->isEnabled()) {
+                return;
+            }
+
+            $this->sendPayload(
+                $this->handleRequest($event->request)
+            );
+        });
+
+        Event::listen(ResponseReceived::class, function (ResponseReceived $event) {
+            if (!$this->isEnabled()) {
+                return;
+            }
+
+            $this->sendPayload(
+                $this->handleResponse($event->request, $event->response)
+            );
+        });
+    }
+
+    public function enable(string $label = null): void
+    {
+        $this->label = $label;
+
+        $this->enabled = true;
+    }
+
+    public function disable(): void
+    {
+        $this->enabled = false;
+    }
+
+    public function isEnabled(): bool
+    {
+        $this->trace = array_slice($this->findSource(), 0, 5)[0] ?? [];
+
+        if (!boolval(config('laradumps.send_http_client'))) {
+            return $this->enabled;
+        }
+
+        return true;
+    }
+
+    public function setTrace(array $trace): array
+    {
+        return $this->trace = $trace;
+    }
+
+    protected function getRequestType(Request $request): string
+    {
+        if ($request->isJson()) {
+            return 'Json';
+        }
+
+        if ($request->isMultipart()) {
+            return 'Multipart';
+        }
+
+        return 'Form';
+    }
+
+    protected function handleRequest(Request $request): Payload
+    {
+        return new TableV2Payload([
```

Comment:
Can't use the current table? We need to define a single view type here. We will have to think about the future maintenance of this method

Reply:
In current `table` there is `thead` but in this (`table-v2`), I only used `tbody` where first `td` acts as `head`;

I think its useful where we've to show data in `key => value` format;

Or may be you can suggest better option... [as I'm dumb in UI :)]